sdk/containerregistry/azure-containers-containerregistry/swagger/autorest.md

```diff
@@ -40,6 +40,6 @@ add-context-parameter: true
 context-client-method-parameter: true
 service-interface-as-public: true 
 models-subpackage: implementation.models
-custom-types: AcrErrorsException,DeleteRepositoryResult,AcrErrors
+custom-types: ContentProperties
```

Comment:
I saw more than this in models using the code generated file header, are there classes missing here?

Reply:
All the other models in the public surface area are hand authored as they are immutable types. 

---
In reply to: [609169983](https://github.com/Azure/azure-sdk-for-java/pull/20368#discussion_r609169983) [](ancestors = 609169983)